src/promptflow/promptflow/_sdk/_tracing.py

```diff
@@ -86,13 +86,13 @@ def _print_tracing_url_from_local(
     exp: typing.Optional[str] = None,
     run: typing.Optional[str] = None,
 ) -> None:
-    url = f"http://localhost:{pfs_port}/v1.0/ui/traces"
+    url = f"http://localhost:{pfs_port}/v1.0/ui/traces/"
     if run is not None:
-        url += f"?run={run}"
+        url += f"?#run={run}"
```

Comment:
why need this extra #?

Reply:
this should be the way how UX/React route

Reply:
double confirmed with Kaiyi, you can regard this style with `#` is the way how front end manage the route, so we might need this breaking change.

Reply:
good to know, can  add the explanation to pr description.